From 5b418757f452ab9df4854027fa4ee0d6a6482135 Mon Sep 17 00:00:00 2001
Message-ID: <5b418757f452ab9df4854027fa4ee0d6a6482135.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 00:15:41 +0100
Subject: [PATCH 06/19] gnu: go-github-com-jacobsa-oglematchers: Move to (gnu
 packages golang-check).

* gnu/packages/golang.scm (go-github-com-jacobsa-oglematchers): Move
from here...
* gnu/packages/golang-check.scm: ... to here.
---
 gnu/packages/golang-check.scm | 28 ++++++++++++++++++++++++++++
 gnu/packages/golang.scm       | 28 ----------------------------
 2 files changed, 28 insertions(+), 28 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 81583a5ac7..8c6bf03f74 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -38,6 +38,34 @@ (define-module (gnu packages golang-check)
 ;;;
 ;;; Code:
 
+(define-public go-github-com-jacobsa-oglematchers
+  (let ((commit "141901ea67cd4769c6800aa7bfdfc558fa22bda5")
+        (revision "0"))
+    (package
+      (name "go-github-com-jacobsa-oglematchers")
+      (version (git-version "0.0.0" revision commit))
+      (source (origin
+                (method git-fetch)
+                (uri (git-reference
+                      (url "https://github.com/jacobsa/oglematchers")
+                      (commit commit)))
+                (file-name (git-file-name name version))
+                (sha256
+                 (base32
+                  "09ff5x6vbhd9zl1z4yzyk573ifh16rry38q1rx986kbz4hqkmniq"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "github.com/jacobsa/oglematchers"
+         ;; break loop with with go-github-com-jacobsa-ogletest
+         #:tests? #f))
+      (home-page "https://github.com/jacobsa/oglematchers")
+      (synopsis "Matchers for Go testing framework")
+      (description
+       "Package oglematchers provides a set of matchers useful in a testing or mocking
+framework.  These matchers are inspired by and mostly compatible with Google
+Test for C++ and Google JS Test.")
+      (license license:asl2.0))))
+
 (define-public go-github-com-jacobsa-oglemock
   (let ((commit "e94d794d06ffc6de42cb19d0dab3c219efdd6dcf")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 794e5f0f49..afc1a9bd30 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -1311,34 +1311,6 @@ (define-public go-github-com-aperturerobotics-jacobsa-crypto
 not included in the Go standard library.")
       (license license:asl2.0))))
 
-(define-public go-github-com-jacobsa-oglematchers
-  (let ((commit "141901ea67cd4769c6800aa7bfdfc558fa22bda5")
-        (revision "0"))
-    (package
-      (name "go-github-com-jacobsa-oglematchers")
-      (version (git-version "0.0.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://github.com/jacobsa/oglematchers")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "09ff5x6vbhd9zl1z4yzyk573ifh16rry38q1rx986kbz4hqkmniq"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/jacobsa/oglematchers"
-         ;; break loop with with go-github-com-jacobsa-ogletest
-         #:tests? #f))
-      (home-page "https://github.com/jacobsa/oglematchers")
-      (synopsis "Matchers for Go testing framework")
-      (description
-       "Package oglematchers provides a set of matchers useful in a testing or mocking
-framework.  These matchers are inspired by and mostly compatible with Google
-Test for C++ and Google JS Test.")
-      (license license:asl2.0))))
-
 (define-public go-github-com-jacobsa-ogletest
   (let ((commit "80d50a735a1108a2aeb7abc4a988d183f20c5292")
         (revision "0"))
-- 
2.41.0

